### ISSUE TITLE
Impossible to unshared an item

### ISSUE THREAD

**Issue by LapinFou** [bitwarden/mobile#597]
I got the Premium & the Families subscriptions.
I got 3 collections: one for each of my 2 kids + one with my wife.

Last night, my son (away of our house) asked me our Netflix credentials.
So, I edited the concerned item and shares it through the collection button. BitWarden is so cool and handy. 😄 
Then, when my soon was done, I decided to "unshared" this item.
Unfortunately this didn't work! I kept having an error message telling me than I must select at least one collection.
This issue is the same on **iOS**/**Android**/**Firefox plugin**/**Windows Desktop** apps and the **main Web panel**.
The only way I found to unshared this item was through the sub-menu "_Organization_" from the Web interface.

**Comment by LapinFou** [comment]
In others words, I cannot unshared an item from an organization. It must be done from the organization menu on the Web interface.

**Comment by LapinFou** [comment]
I have done some test with another item. Same issue. Moreover I can unselected it from all collection, but the share icon is still display.
So, looks like after adding an item in the organization, you can't remove it! I'm quite sure it wasn't the case when I starting to use BitWarden few months ago. Could be a regression since the last update?!

**Comment by kspearrin** [comment]
This has always been the case. Once ownership of an item is transferred to an organization, it cannot be transferred back to a user account. The only workaround is to re-create the item and then delete it from the org.

See https://community.bitwarden.com/t/add-unshare-option/604

**Comment by LapinFou** [comment]
@kspearrin 
OK. Thanks. I missed this point. Now, I understood that it is an ownership transfer, not a "sharing".

However, why the shared item MUST be part of a collection?
From the Web interface, I can choose "none" through the **unselected all** button, but this doesn't work on the apps.
The only way to "Unassigned" an item transferred in the organization is from the Web GUI. It is not possible to do it from the apps.

**Comment by kspearrin** [comment]
@LapinFou This is because you are doing this from the organization admin portal, which gives you additional rights as the org admin, like orphaning an item from collections.

**Comment by LapinFou** [comment]
OK, this makes sense. However, since I'm the owner of the organization, I was expecting to be able to do it from the apps.
But, I think, I got what you said. The organization portal is not available on standard apps.
I think, this should be better explain in the FAQ. 
And, what about a "duplicate" button? Should I opened a new ticket for that? Or is it already in the pipe?

PS: since I discovered BitWarden, I'm talking about it to all my friends. You have done a great peace of software. Very flexible, cheap with many features. Keep going! 💯
